docs/index.rst

```diff
@@ -151,6 +151,7 @@ Contents:
    html-templates
    client
    logging
+   typing-indicators
```

Comment:
We should also add links to where this will live on RTD in comments at the top of all the typing-indicator specific files, so that it's easy to find.